src/utils/formHelperFunctions.ts

```diff
@@ -504,3 +506,34 @@ export function isStepTerminal(step: any) {
 
   return !hasNext;
 }
+
+export function saveUrlParams(
+  updateFieldValues: (newFieldValues: any, rerender?: boolean) => boolean,
+  client: Client
+) {
+  const params = new URLSearchParams(featheryWindow().location.search);
+  const hiddenFields: any = {};
+  params.forEach((value, key) => {
+    if (['feathery_1', 'feathery_2', '_slug'].includes(key)) return;
+    hiddenFields[key] = value;
+  });
+  if (Object.getOwnPropertyNames(hiddenFields).length === 0) return;
```

Comment:
what is this check for? and if we want to short circuit it, could we just short circuit by checking params above?

Reply:
hosted forms can have the _slug query param so I only wanted to proceed if there were other query params

this seems like the most direct way to do that, rather than needing a separate check for _slug (and the others) above

Reply:
ah that makes sense